x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/single_page_layout/index.tsx

```diff
@@ -24,6 +24,7 @@ import type { EuiStepProps } from '@elastic/eui/src/components/steps/step';
 import { safeLoad } from 'js-yaml';
 
 import { useCancelAddPackagePolicy, useOnSaveNavigate } from '../hooks';
+import type { CreatePackagePolicyRequest } from '../../../../../../../common';
```

Comment:
(nit) Add a newline between imports from the same relative directory (same number of `..`)